package.json

```diff
@@ -15,22 +15,24 @@
   },
   "main": "lib/kv-redis.js",
   "scripts": {
-    "test": "mocha test/integration/*.js",
+    "test": "mocha test/unit/*.js && mocha test/integration/*.js",
```

Comment:
> By listing them as `mocha test/unit/*.js test/integration/*.js` won't it test in that order? You could add `--bail` to get fail-fast, as well.
> I believe we should get the same effect by running `mocha test/unit/*.js test/integration.js`.

Interesting, it seems that this approach works. I'll use it then!

> You could add `--bail` to get fail-fast, as well.

I prefer to see all test failures, it usually helps to understand whether it's a problem in the code (a single test failing) or perhaps something else (many tests failing because of CI environment problem).